### DISCUSSION TITLE
What is product type used for?

### DISCUSSION THREAD

**Discussion by winston0410** [medusajs/medusa#4478] [Q&A]
From the doc I can see multiple ways to group products
- categories: to group products in a hierarchy
- tags: to group product without hierarchy
- collection: seems to be overlapping with tags??, but a flat hierarchy as well

And I am not sure about [product type](https://docs.medusajs.com/api/store#tag/Product-Types), as it is not mentioned by the doc, what should it be used for, and what problem can it solve that other way of grouping cannot?

**Comment by srindom** [comment]
@winston0410 - thanks for asking. 

You are right that there is some overlap between the different product fields, but having these gives some optionality when mapping your product structure. 

Product Type specifically was added with the intention that it could be a discriminator for different product detail page implementations. E.g., you might have different layouts for a shirt vs. a digital product page, and the type allows you to choose the right one.

**Reply by winston0410** [reply]
Thank you so much for your reply, and that make sense. I also notice that product type is also mentioned for calculating tax, does that mean that it works as the junction table for grouping products with tax, and which other groupings(i.e. categories, collections, tags) does not do that?
